newsletters/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils import timezone
+
+
+class department(models.Model):
+    '''
+    Stores the detail of department linked with its newsletters
+    '''
+    department = models.CharField(max_length=50,
+                                  blank=False,
+                                  null=False)
```

Comment:
I don't think you need this explicitly.

Reply:
Sir, while developing this newsletter section, newsletters were categorized by their respective departments.

Reply:
Okay sir. I'll merge this into newsletters model itself.